feat(replay): Add Troubleshooting item for "clicks in wrong places" (#12782)
getsentry/sentry-docs: docs/platforms/javascript/common/session-replay/troubleshooting.mdx

If you want to turn off [rage click issues](/product/issues/issue-details/replay-issues/) altogether, you can disable them by going to the **Settings** page in Sentry, clicking "Projects", and selecting the project for which you want to disable rage clicks. From here, select "Replays" under the "PROCESSING" heading and toggle off "Create Rage Click Issues".

</Expandable>

<Expandable title="Clicks appear in the wrong location" permalink>

> Clicks in replays can appear in the incorrect location due to rendering differences between your application and the replayer. A common case of this is with text rendering: custom fonts are not supported in the replayer and can cause the height of elements to differ between production and the replay. This means that the scroll position being incorrect and results in mouse clicks appearing to click the wrong location. Text masking of variable-width fonts can also cause this.

Reviewer: ```suggestion
Clicks in replays can appear in the incorrect location due to rendering differences between your application and the replayer. A common case of this is with text rendering: custom fonts are not supported in the replayer and can cause the height of elements to differ between production and the replay. This means that the scroll position will be incorrect and will result in mouse clicks appearing in the wrong location. Text masking of variable-width fonts can also cause this.
```

> Text masking of variable-width fonts can also cause this because the size of the masking text ("*") will not necessarily have the same dimensions as the original character it replaced. Unfortunately, there is currently no workaround: please [follow this GitHub Issue](https://github.com/getsentry/sentry-javascript/issues/15449) for updates.

Reviewer: ```suggestion
Text masking of variable-width fonts can also cause this because the size of the masking text ("*") will not necessarily have the same dimensions as the original character it replaced. There is currently no workaround; please [follow this GitHub Issue](https://github.com/getsentry/sentry-javascript/issues/15449) for updates.
```
